class.jetpack-settings.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/**
+ * All the Settings we want to sync to .com
+ *
+ * This is also used when displaying settings in the API
+ */
+class Jetpack_Settings {
+
+	public static $settings = array(
+		// also exists as "options"
+		'default_ping_status'     => 'bool',
+		'default_comment_status'  => 'bool',
+
+		// new stuff starts here
```

Comment:
No need to include this comment. As soon as this is merged these are no longer "new" :)